scripts/ps_keyring_plugins_test/ps_keyring_plugins_test_81.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+set -e
+
+if [ -f /etc/mysql/my.cnf ]; then
+  MYCNF="/etc/mysql/my.cnf"
+else
+  MYCNF="/etc/my.cnf"
+fi
+
+LOG="/tmp/keyring_plugins_test_run.log"
+echo -n > ${LOG}
+
+if [ -z "$1" ]; then
+  echo "This script needs parameter ps57|ps80|ps81"
+  exit 1
+elif [ "$1" != "ps57" -a "$1" != "ps80" -a "$1" != "ps81" ]; then
+  echo "Version not recognized!"
+  exit 1
+#else
+#  VERSION="$1"
+fi
+
```

Comment:
If this is component test, it would be nice to check that the keyring component is present in components list (if it should be there)